common/lib/xmodule/xmodule/js/src/problem/edit.js

```diff
@@ -1,63 +1,76 @@
-// Generated by CoffeeScript 1.6.1
+/* global CodeMirror, _, XModule, PrettyPrint */
+/* eslint no-useless-escape: 0 */
```

Comment:
This is for the regexp statements in the "toXML" method. When I tried deleting "useless escapes", I got some unit test failures related to the markdown -> XML conversion (though only in Jenkins, so it must be browser-specific). So I'm going to leave them all is-is.